api/src/main/resources/workbench.yaml

```diff
@@ -1869,6 +1871,11 @@ definitions:
         description: The email with which to contact the bug reporter.
         type: string
 
+  BugReportType:
+    type: string
+    description: Possible value for bug reports
+    enum: [APPLICATION, DATA]
```

Comment:
Do we want to also add an attribute for environment reported from?  either here or above? Think it would be a good pc of information to have included.